external/anomaly/ote_anomalib/task.py

```diff
@@ -221,13 +221,13 @@ def infer(self, dataset: DatasetEntity, inference_parameters: InferenceParameter
         self.trainer.predict(model=self.model, datamodule=datamodule)
         return dataset
 
-    def evaluate(self, output_resultset: ResultSetEntity, evaluation_metric: Optional[str] = None) -> None:
+    def evaluate(self, output_resultset: ResultSetEntity, _evaluation_metric: Optional[str] = None) -> None:
```

Comment:
why do you rename it? It does not follow the interface anymore

Reply:
@Ilya-Krylov, if not configured otherwise, pylint throws an error when the method argument is not used. An underscore as a prefix would inform pylint that the argument won't be used. Pylint therefore wouldn't raise an error.

This might be ignored in pylintrc, which would be the reason why this previously wouldn't cause a problem.

Reply:
yes, this was renamed to please pylint